### PR TITLE
Skip systemctl status test for NUC

### PR DESCRIPTION
**Pull request by azbeleva**
Skip test in case if test fails because issue is known but has low priority.

### CODE REVIEW

File: Robot-Framework/test-suites/bat-tests/others.robot [hunk 1]
```diff
@@ -34,11 +35,19 @@ Check QSPI version
 
 Check systemctl status
     [Documentation]    Verify systemctl status is running
-    [Tags]             bat  SP-T104  nuc  orin-agx  orin-nx  riscv
+    [Tags]             bat  SP-T104  orin-agx  orin-nx  riscv
     [Setup]     Connect
     Verify Systemctl status
     [Teardown]  Close All Connections
 
+Check systemctl status on NUC
```
[the remark this answers is not in the document]
Comment (azbeleva) [reply Robot-Framework/test-suites/bat-tests/others.robot:43]: Currently ${DEVICE} can contain any name which will given to exact NUC in setup, not just "NUC"

[the remark this answers is not in the document]
Comment (maaharko) [reply Robot-Framework/test-suites/bat-tests/others.robot:43]: Ok, I see. Then maybe use 
`IF  "NUC" in ${DEVICE}`

I'm just thinking that it is not so good idea to create separate tests for separate devices

File: Robot-Framework/test-suites/bat-tests/others.robot [hunk 2]
```diff
@@ -6,6 +6,7 @@ Documentation       Common system tests
 Resource            ../../resources/ssh_keywords.resource
 Resource            ../../resources/serial_keywords.resource
 Library             ../../lib/output_parser.py
+Library             BuiltIn
```
[the remark this answers is not in the document]
Comment (azbeleva) [reply Robot-Framework/test-suites/bat-tests/others.robot]: not necessary, you're right, removed

File: Robot-Framework/test-suites/bat-tests/others.robot (hunk 1 above)
[the remark this answers is not in the document]
Comment (azbeleva) [reply Robot-Framework/test-suites/bat-tests/others.robot]: Fixed

**Review by maaharko — APPROVED** [review]

**Pull request merged**